Dockerfile

```diff
@@ -6,5 +6,5 @@ COPY . /go/src/github.com/luizalabs/teresa-api
 
 RUN make build-server
 
-CMD ["./teresa-server", "run", "--port", "8080"]
-EXPOSE 8080
+ENTRYPOINT ["./teresa-server"]
```

Comment:
Right. I guess, its considered a docker [best practice ](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#entrypoint), but the best thing is that you don't have to rebuild the image (you may have to pass a `--debug` flag in the future).